sim/death_knight/death_strike.go

```diff
@@ -39,7 +39,7 @@ func (dk *DeathKnight) registerDeathStrikeSpell() {
 		ClassSpellMask: DeathKnightSpellDeathStrikeHeal,
 
 		DamageMultiplier: 1,
-		ThreatMultiplier: 1,
```

Comment:
Healing usually has a threat modifier. Is this change confirmed?

Reply:
@rosenrusinov This was posted by Char here in our Discord, you can check with him; https://discord.com/channels/891730968493305867/909597996449157160/1271941989214191737
Or should I in that case put it to 0 if there's no threat for it?